src/components/about/AboutRouter.js

```diff
@@ -0,0 +1,19 @@
+import { Route } from 'react-router-dom'
+
+import ImportedDatasetsPage from '../about/ImportedDatasetsPage'
+import ProjectPage from './ProjectPage'
+
+const pages = [
+  {
+    path: '/about/project',
+    component: ProjectPage,
+  },
+  {
+    path: '/about/dataset',
+    component: ImportedDatasetsPage,
+  },
+]
+
+const AboutRouter = pages.map((props) => <Route key={props.path} {...props} />)
```

Comment:
This isn't a component, so should be lowercase. `aboutRoutes`, not `AboutRouter`